### PR TITLE
Feather m4 can

### PR DESCRIPTION
**Pull request by jepler**
.. product link does not exist yet, feather m4's was used.  That's why I'm leaving this in draft status.

The big dump of files is from http://packs.download.atmel.com/ http://packs.download.atmel.com/Atmel.SAME51_DFP.1.1.139.atpack

### CODE REVIEW

**Comment by jepler** [comment]
@ladyada in case you want to pick up the bootloader from the PR here it is

File: boards/feather_m4_can/board_config.h [hunk 1]
```diff
@@ -0,0 +1,29 @@
+#ifndef BOARD_CONFIG_H
+#define BOARD_CONFIG_H
+
+#define VENDOR_NAME "Adafruit Industries"
+#define PRODUCT_NAME "Feather M4 Express"
```
Comment (ladyada) [inline boards/feather_m4_can/board_config.h:5]: change to Feather M4 CAN Express

File: boards/feather_m4_can/board_config.h [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+#define VOLUME_LABEL "FTHRCANBOOT"
+#define INDEX_URL "http://adafru.it/3857"
```
Comment (ladyada) [inline boards/feather_m4_can/board_config.h:7]: fix url

**Review by ladyada — CHANGES_REQUESTED** [review]

**Comment by jepler** [comment]
@ladyada updated!

**Review by ladyada — APPROVED** [review]
ok!

**Comment by ladyada** [comment]
looks good, can merge!

**Comment by jepler** [comment]
whoops, it's marked ready for review now!  I don't think I have permissions to merge in this repo

**Pull request merged**